Manatee.Trello/Rest/WebApiClient.cs

```diff
@@ -176,11 +177,15 @@ private static HttpContent GetContent(WebApiRestRequest request)
 					formData.Add(content, $"\"{parameter.Key}\"");
 				}
 
-				var byteContent = new ByteArrayContent(request.File);
-				formData.Add(byteContent, "\"file\"", $"\"{request.FileName}\"");
-				TrelloConfiguration.Log.Debug($"\tContent: {formData}");
 
-				return formData;
+                var fileStream = File.Open(request.File, FileMode.Open);
+                var fileInfo = new FileInfo(request.File);
+                formData.Add(new StreamContent(fileStream), "\"file\"", string.Format("\"{0}\"", "as" + fileInfo.Extension));
+                formData.Add(new StringContent("mimeType"), "image/png");
```

Comment:
This is probably the bit I'm missing.  I can add a `mimeType` parameter to the method to pass here.  I think once Trello has the mime type, it'll know what to do with it.  Still, I'm not seeing any problems with the current implementation.

Reply:
I believe that will be right